js/semver.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// eslint-disable-next-line no-var
+var semver = {
```

Comment:
we don't really need a separate script in manifest and a separate variable just for this, it should be merged into usercss.js within `usercss` object.